_episodes/16-resources.md

```diff
@@ -39,16 +39,16 @@ think your job will need.
 > {: .bash}
 > 
 > The `fastqc` command is provided by the `fastqc` module. You'll need to figure out a good amount
-> of resources to ask for for this first "test run". You might also want to have the scheduler email
-> you to tell you when the job is done.
+> of resources to ask for for this first "test run". You might also want to have the scheduler 
+> email  you to tell you when the job is done.
```

Comment:
extraneous space

Reply:
Thank you for the review, fixed in 4a95a7e